Gemfile.lock

```diff
@@ -1,24 +1,23 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (5.2.4.3)
```

Comment:
not sure if we need the `Gemfile.lock` locally, but grabbed from https://github.com/ministryofjustice/cloud-platform-user-guide/blob/main/Gemfile.lock anyway